platform/core/src/extensions/MODULE_TYPES.js

```diff
@@ -1,5 +1,6 @@
 export default {
   COMMANDS: 'commandsModule',
+  UI_CUSTOMIZATION: 'customizationModule',
```

Comment:
This is confusing, why we are using the `customizationModule` but it is `UI_CUSTOMIZATION`. if you look at all other types here, they all match. Please stick to either `CustomizationModule` and `CustomizatoinService` or `UICustomizationModule` and `UICustomizationService`

Reply:
They don't all match already, but I agree, they probably should match in a well defined fashion.